### PR TITLE
update README

### PR DESCRIPTION
**Pull request by voku**
This change is [Reviewable](https://reviewable.io/reviews/rectorphp/rector/2116)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -89,6 +89,10 @@ vendor/bin/rector process ../project --autoload-file ../project/vendor/autoload.
 
 Or make use of `rector.yaml` config:
 
+```bash
+vendor/bin/rector process ../project --config ../project/rector.yaml
+```
```
Comment (TomasVotruba) [inline README.md]: The line above refers about the confit content, not `--config` option.

Better add use case bellow with example, when this option useful

**Comment by voku** [comment]
@TomasVotruba you are right, so I added some more headlines and refactored many question in statement e.g. ```What can Rector do for You? -> Features``` || ```How to Apply Coding Standards? -> Rector !== Coding Standards```. I also added a simple link list on top and added the project logo. 

-> preview: https://github.com/voku/rector/tree/readme-update#rector

**Comment by TomasVotruba** [comment]
I like current design, the menu is confusing even me.  
You can find tips for writing good readme: https://www.makeareadme.com/

Let's just add `--config` option.

**Comment by voku** [comment]
Was only a suggestion. :-) Add the config example wherever you like. ;-)

## Configuration

#### Config Loading

The config file `rector.yaml` is passed to the rector executable with the `--config` option:

```bash
vendor/bin/rector process ../project --config ../project/rector.yaml
```

**Comment by TomasVotruba** [comment]
That would be great

**Pull request closed without merging**